test/supernode_client/main.go

```diff
@@ -142,6 +142,7 @@ func sizeOfIthFile(i int64) int64 {
 }
 
 func runFileAddingWorker(n *core.IpfsNode) error {
+	errs := make(chan error, math.MaxInt32)
```

Comment:
i believe this thing used to just sit, adding a new file every second. it could just as well loop forever.